task2/script.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+function isArrayOne(arr) {
+  return arr.constructor === Array;
+}
+
+function isArrayTwo(arr) {
+  return arr instanceof Array;
+}
+
+function isArrayThree(arr) {
+  return {}.toString.call(arr) === '[object Array]'
+}
+
+function range() {
+  var arr = [];
+  switch (arguments.length) {
```

Comment:
not cool switch usage, if is better here
you have mostly the same loop, why do you copy past it?